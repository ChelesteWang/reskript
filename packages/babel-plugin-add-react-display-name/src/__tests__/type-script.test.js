const runSuite = require('./suite');

runSuite('type-script');
