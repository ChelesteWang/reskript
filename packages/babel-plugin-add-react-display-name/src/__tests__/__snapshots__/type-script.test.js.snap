// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`type script export default arrow function as a module's index 1`] = `
"const _DefaultExport = () => /*#__PURE__*/React.createElement(\\"div\\", null);

// export default arrow function as a module's index

/* eslint-disable */
export default _DefaultExport;
_DefaultExport.displayName = \\"DefaultExport\\";"
`;
